### DISCUSSION TITLE
Customising component styles?

### DISCUSSION THREAD

**Discussion by jacksleight** [filamentphp/filament#1366] [Help] [answered]
Is there a way to target specific components with custom styling? Things like the sidebar, modals or breadcrumbs? I know you can override the view files, but that feels a bit overkill for minor styling tweaks, and could become a headache to maintain if things get out of sync when you update Filament. A custom Tailwind config is great for global things like fonts and colours, but can’t really help with specific components.

I was just wondering if there’s something I’ve missed? If not, would adding some extra styling hooks/semantic classes to the components be considered?

**Comment by danharrin** [comment, marked as answer]
We have a PR open here - #1291

**Reply by jacksleight** [reply]
Amazing! :-)
